src/Forms/ConfirmedPasswordField.php

```diff
@@ -43,7 +43,8 @@ class ConfirmedPasswordField extends FormField
     public $requireStrongPassword = false;
 
     /**
-     * Allow empty fields in serverside validation
+     * Allow empty fields in when entering the password for the first time
+     * If this is set to true then a random password will be created before saving
```

Comment:
```suggestion
     * If this is set to true then a random password will be created before saving new records
```
This only applies if it's a new record.

Also, is this true for _any_ use of this field? Or just for `Member` records? If the latter, don't add this note at all as it's not relevant for the form field itself.

Reply:
Have changed to something different

It's relevant for any use of the field